lib/octobox/notifications/sync_subject.rb

```diff
@@ -25,10 +25,20 @@ def update_subject_in_foreground(force = false)
         Subject.sync(remote_subject.to_h.as_json)
       end
 
+      def github_client
+        if user.personal_access_tokens_enabled?
+          user.github_client
+        elsif app_installation.present?
+          user.app_installation_client
+        else
+          user.github_client
+        end
```

Comment:
This methods reads a bit weird, but the logic is correct. I am very open to suggestions about how to make this code clearer.

`github_client` picks between the Personal Access Token and the 'normal' Access Token, so when we call it on line 30 we expect to be using the Personal Access Token, while when we call it on line 34 we are expecting to use the 'normal' Access Token

Reply:
I pulled apart the User methods more so we don't have to rely on just `github_client` here!